src/helpers/api-helpers.php

```diff
@@ -169,4 +169,33 @@ public function log_to_slack( $message ) {
 
 		return json_decode( $result );
 	}
+
+	public function call_wpcom_api( $query, $data, $method = 'POST' ) {
+		$api_request_url = WPCOM_API_ENDPOINT . $query;
+
+		$headers = array(
+			'Accept: application/json',
+			'Content-Type: application/json',
+			'Authorization: Bearer '. WPCOM_API_ACCOUNT_TOKEN,
+			'User-Agent: PHP',
+		);
+
+		$options = array(
+			'http' => array(
+				'header'        => $headers,
+				'ignore_errors' => true,
+			)
+		);
+
+		if ( ! empty( $data ) && in_array( $method, array( 'POST', 'PUT' ) ) ) {
```

Comment:
Do we need to handle the case where the method is `GET` for future uses? The method currently won't be sent in the API call if it's not `POST|PUT`.